system/lib/wasmfs/backends/fetch_backend.cpp

```diff
@@ -13,15 +13,17 @@
 // See library_wasmfs_fetch.js
 
 extern "C" {
-void _wasmfs_create_fetch_backend_js(wasmfs::backend_t);
+void _wasmfs_create_fetch_backend_js(wasmfs::backend_t, char*);
```

Comment:
Should this be `const char *`? (same below for argument to wasmfs_create_fetch_backend)

Reply:
Done.